pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -964,6 +964,14 @@ public List<ThetaSketchAccumulator> extractAggregationResult(AggregationResultHo
   @Override
   public List<ThetaSketchAccumulator> extractGroupByResult(GroupByResultHolder groupByResultHolder, int groupKey) {
     List result = groupByResultHolder.getResult(groupKey);
+    if (result == null) {
```

Comment:
TODO: trying to figure out proper test cases with binary encoding that actually can trigger this codepath.

Reply:
done